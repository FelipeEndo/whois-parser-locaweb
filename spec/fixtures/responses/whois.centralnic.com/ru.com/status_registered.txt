Domain Name: SRK.RU.COM
Registry Domain ID: D450826-CNIC
Registrar WHOIS Server: whois.instra.net
Registrar URL: https://instra.com/
Updated Date: 2021-04-19T15:40:48.0Z
Creation Date: 2006-07-31T10:06:04.0Z
Registry Expiry Date: 2025-07-31T23:59:59.0Z
Registrar: Instra Corporation Pty Ltd
Registrar IANA ID: 1376
Domain Status: ok https://icann.org/epp#ok
Registrant Email: https://whois.centralnic.com/contact/srk.ru.com/registrant
Admin Email: https://whois.centralnic.com/contact/srk.ru.com/admin
Tech Email: https://whois.centralnic.com/contact/srk.ru.com/tech
Name Server: NS7.ZONEEDIT.COM
Name Server: NS11.ZONEEDIT.COM
DNSSEC: unsigned
Billing Email: https://whois.centralnic.com/contact/srk.ru.com/billing
Registrar Abuse Contact Email: compliance@instra.com
Registrar Abuse Contact Phone: +61.397831800
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2021-12-15T16:58:26.0Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

>>> IMPORTANT INFORMATION ABOUT THE DEPLOYMENT OF RDAP: please visit
https://www.centralnic.com/support/rdap <<<

The Whois and RDAP services are provided by CentralNic, and contain
information pertaining to Internet domain names registered by our
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com)

Access to the Whois and RDAP services is rate limited. For more
information, visit https://registrar-console.centralnic.com/pub/whois_guidance.
