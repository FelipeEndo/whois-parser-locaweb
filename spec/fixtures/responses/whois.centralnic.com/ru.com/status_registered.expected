#disclaimer
  %s == "The Whois and RDAP services are provided by CentralNic, and contain information pertaining to Internet domain names registered by our our customers. By using this service you are agreeing (1) not to use any information presented here for any purpose other than determining ownership of domain names, (2) not to store or reproduce this data in any way, (3) not to use any high-volume, automated, electronic processes to obtain data from this service. Abuse of this service is monitored and actions in contravention of these terms will result in being permanently blacklisted. All data is (c) CentralNic Ltd (https://www.centralnic.com) Access to the Whois and RDAP services is rate limited. For more information, visit https://registrar-console.centralnic.com/pub/whois_guidance."

#domain
  %s == "srk.ru.com"

#domain_id
  %s == "D450826-CNIC"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2006-07-31 10:06:04 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2021-04-19 15:40:48 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2025-07-31 23:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "1376"
  %s.name         == "Instra Corporation Pty Ltd"
  %s.url          == "https://instra.com/"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].email         == "https://whois.centralnic.com/contact/srk.ru.com/registrant"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].email         == "https://whois.centralnic.com/contact/srk.ru.com/admin"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].email         == "https://whois.centralnic.com/contact/srk.ru.com/tech"


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns7.zoneedit.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns11.zoneedit.com"
